### PR TITLE
Interim/windows login module

### PR DESCRIPTION
**Pull request by eekodeerder**
This is interim code which fixes the problem with the WindowsLoginModule not being compatible with JAAS or Tomcat, but does not remove the incorrect modification (for WildFly up to version 10), which broke the compatibility with JAAS and Tomcat.

### CODE REVIEW

File: Source/JNA/waffle-tests/src/test/java/waffle/jaas/WindowsLoginModuleTests.java [hunk 1]
```diff
@@ -291,17 +275,10 @@ void testRoleUnique() throws LoginException {
...
-        for (final Principal principal : subject.getPrincipals()) {
-            if (principal instanceof GroupPrincipal) {
-                int size = Collections.list(((GroupPrincipal) principal).members()).size();
-                Assertions.assertEquals(3, size);
-                Assertions.assertTrue(((GroupPrincipal) principal).isMember(new RolePrincipal("Everyone")));
-                Assertions.assertTrue(((GroupPrincipal) principal).isMember(new RolePrincipal("Users")));
-                Assertions.assertTrue(((GroupPrincipal) principal).isMember(new RolePrincipal("Group 1")));
-            }
-        }
+        Assertions.assertTrue(4, subject.getPrincipals().size() >= 4);
```
Comment (hazendaz) [inline Source/JNA/waffle-tests/src/test/java/waffle/jaas/WindowsLoginModuleTests.java:278]: @eekodeerder Can you fix line 278 here?  Needs the 4 on left side removed.  I think everything will work then.

**Comment by eekodeerder** [comment]
For some reason it is now failing Tomcat 10 build with JDK 8 and windows 2016.

File: Source/JNA/waffle-tests/src/test/java/waffle/jaas/WindowsLoginModuleTests.java (hunk 1 above)
Comment (eekodeerder) [reply Source/JNA/waffle-tests/src/test/java/waffle/jaas/WindowsLoginModuleTests.java]: Sorry.  I booked it in (under draft PR).  Intntion is/was to come back and fix errors as soon as I get time.  I also fixed the test that was failing, but for some reason Tomcat 10 with JDK 8 and windows-2016 is failing to build.

**Comment by hazendaz** [comment]
intermittint issue on actions builds, built again and it worked.  Merging now.

**Pull request merged**